### PR TITLE
Add event media

### PR DESCRIPTION
**Pull request by mjpost**
This PR enriches the `<event>` block, supporting adding event-related data such as the handbook (#298).

It builds on existing parsing routines in an attempt to be general.

### CODE REVIEW

**Comment by mjpost** [comment]
Here's what this looks like now:

image

Need to fix the capitalization, obviously. 

This is a step in the direction of having a more distinct event page. Compare to [the current ACL 2022 event page](https://aclanthology.org/events/acl-2022/).

**Comment by mjpost** [comment]
Link [now works](https://preview.aclanthology.org/event-data/events/acl-2022/).

**Comment by mjpost** [comment]
Noting previous attempt at #918.

**Review by mbollmann — APPROVED** [review]
LGTM, as far as I can tell (the many changes in master have made it a bit hard to follow...).

We could maybe consider marking external links with an extra icon, like `<i class="fas fa-external-link-alt"></i>`.

**Comment by mjpost** [comment]
Note @anthology-assist: Instead of

```
<event>
  <colocated>volume-id</colocated>
  ...
</event>
```

we now do

```
<event>
  <colocated>
    <volume-id>volume-id</volume-id>
    ...
  </colocated>
</event>
```

so you'll have to update the ingestion script(s).
